app/models/grda_warehouse/tasks/identify_duplicates.rb

```diff
@@ -12,9 +12,6 @@ class IdentifyDuplicates
     def initialize(run_post_processing: true)
       setup_notifier('IdentifyDuplicates')
       @run_post_processing = run_post_processing
-      super()
```

Comment:
Unless there is some magic I'm unaware of, there is no superclass so this is pointless. Am I missing something?